lib/srv/sess.go

```diff
@@ -703,6 +704,10 @@ func newSession(id rsession.ID, r *SessionRegistry, ctx *ServerContext) (*sessio
 
 	err = sess.trackerCreate(ctx.Identity.TeleportUser, policySets)
 	if err != nil {
+		if trace.IsNotImplemented(err) {
+			return nil, trace.NotImplemented("Attempted to use Moderated Sessions with a v8 auth server but requires v9.")
```

Comment:
nit: technically we don't know that the auth server is v8, it could be even older, right?

Would it make sense to rephrase the error message? Something like?

```
Moderated sessions requires an Auth Server running Teleport v9 or newer.
```

Reply:
Good point, how does `Attempted to use Moderated Sessions with an Auth Server below the minimum version of 9.0.0` sound?

Reply:
Works for me 👍